coaster/utils/classes.py

```diff
@@ -211,6 +217,10 @@ def value_for(cls, name):
             if isinstance(value, NameTitle) and value.name == name:
                 return key
 
+    @classmethod
+    def nametitles(cls):
+        return list((name, title) for name, title in cls.values())
```

Comment:
What happens if we return a generator instead of a list?

`return ((name, title) for name, title in cls.values())`

Reply:
That works. Just have to convert it to a list in the test.

Reply:
Also, doesn't look like wtforms supports generator for choices https://github.com/wtforms/wtforms/issues/339

Reply:
A list it is, then.